#! /usr/bin/env python

PACKAGE='wp_control'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name       Type      Level Description     Default Min   Max
gen.add("pGain", double_t,    0,    "Proportional gain", 0.5, 0.5, 0.5)
gen.add("gpsHeading",       bool_t,    0,    "True to use gps heading, false for fused", False)
gen.add("wpRadius",       double_t,    0,    "Distance from waypoint to move to next", 2,     0, 50)
gen.add("wpnum",       double_t,    0,    "Distance from waypoint to move to next", 2,     0, 50)

exit(gen.generate(PACKAGE, "gpsWaypoint", "gpsWaypoint_params"))
